libs/hdf-converters/src/sonarqube-mapper.ts

```diff
@@ -130,6 +130,7 @@ export class SonarQubeResults {
           params: {
             componentKeys: this.projectId,
             types: 'VULNERABILITY',
+            statuses: 'OPEN,REOPENED',
```

Comment:
```
Statuses

After creation, issues flow through a lifecycle, taking one of the following statuses:

    Open - set by SonarQube on new issues
    Confirmed - set manually to indicate that the issue is valid
    Resolved - set manually to indicate that the next analysis should Close the issue
    Reopened - set automatically by SonarQube when a Resolved issue hasn't actually been corrected
    Closed - set automatically by SonarQube for automatically created issues.
```

Shouldn't we actually be looking at all of these statuses aside from 'Closed'?

Reply:
Good call! Confirmed and  use case i have that cause this was only closed issues.